src/scripts/main.js

```diff
@@ -1 +1,38 @@
 'use strict';
+
+const navBtn = document.querySelector('#nav-menu-trigger');
```

Comment:
```suggestion
const navButton = document.querySelector('#nav-menu-trigger');
```
Let's use the `class` selector all the time. Let's leave the `id` only for page navigation